src/Utility/Window.cpp

```diff
@@ -23,7 +25,15 @@
 
 namespace Utility {
 
-Environment::Environment(Window *view) : QObject(view), m_view(view) {}
+Environment::Environment(Window *view)
+    : QObject(view),
+      m_view(view),
+      m_minimapOnLeft(true),
+      m_playerIndicatorColor(QColor::fromRgb(255, 0, 0)) {
+  m_playerIndicatorTimer.setInterval(50);
+  connect(&m_playerIndicatorTimer, SIGNAL(timeout()), this, SIGNAL(playerLocationChanged()));
```

Comment:
jest przyjazna skladnia connecta bez tych gownianych makr
connect(&m_playerIndicatorTimer, &QTimer::timeout, this &Environment::playerLocationChanged)